e2e-tests/cypress/e2e/journeys/insurance/your-buyer/your-buyer.spec.js

```diff
@@ -0,0 +1,124 @@
+import { ROUTES } from '../../../../../constants';
+import { INSURANCE_ROOT } from '../../../../../constants/routes/insurance';
+import getReferenceNumber from '../../../helpers/get-reference-number';
+
+const { START } = ROUTES.INSURANCE;
+const insuranceStartRoute = START;
+
+const CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER_DETAILS;
+
+const goToPageDirectly = (referenceNumber) => {
+  cy.visit(`${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.YOUR_BUYER.YOUR_BUYER_DETAILS}`, {
+    auth: {
+      username: Cypress.config('basicAuthKey'),
+      password: Cypress.config('basicAuthSecret'),
+    },
+  });
+};
+
+context('Insurance - Your Buyer - Type of your buyer Page - As an exporter, I want to enter the buyer details', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.YOUR_BUYER.YOUR_BUYER_DETAILS}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  it('passes the audits', () => {
+    cy.lighthouse({
+      accessibility: 100,
+      performance: 75,
+      'best-practices': 100,
+      seo: 70,
+    });
+  });
+
+  it('should render a header with href to insurance start', () => {
+    partials.header.serviceName().should('have.attr', 'href', insuranceStartRoute);
+  });
+
+  it('renders a back link with correct url', () => {
+    partials.backLink().should('exist');
+    partials.backLink().invoke('text').then((text) => {
+      expect(text.trim()).equal(LINKS.BACK);
+    });
+
+    partials.backLink().click();
+
+    const expectedUrl = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.YOUR_BUYER.YOUR_BUYER_DETAILS}`;
+
+    cy.url().should('eq', expectedUrl);
+
+    goToPageDirectly(referenceNumber);
+  });
+
+  it('renders an analytics cookies consent banner that can be accepted', () => {
+    cy.checkAnalyticsCookiesConsentAndAccept();
+  });
+
+  it('renders an analytics cookies consent banner that can be rejected', () => {
+    cy.rejectAnalyticsCookies();
+  });
+
+  it('renders a phase banner', () => {
+    cy.checkPhaseBanner();
+  });
+
+  it('renders a page title and heading with caption', () => {
+    const expectedPageTitle = `${CONTENT_STRINGS.PAGE_TITLE} - ${ORGANISATION}`;
+    cy.title().should('eq', expectedPageTitle);
+
+    headingCaption().invoke('text').then((text) => {
+      expect(text.trim()).equal(CONTENT_STRINGS.HEADING_CAPTION);
+    });
+
+    heading().invoke('text').then((text) => {
+      expect(text.trim()).equal(CONTENT_STRINGS.PAGE_TITLE);
+    });
+  });
+
```

Comment:
Looks there isn't a page title or heading in the page yet?

Can we add a test for the countries field and label is rendering please? Like in here https://github.com/UK-Export-Finance/exip/blob/main-application/e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/single-contract-policy/single-contract-policy.spec.js#L220 but we don't need to mention "supported" countries as all countries will be rendered